### ISSUE TITLE
Mapped field name

### ISSUE THREAD

**Issue by erlanggadewa** [notiz-dev/prisma-dbml-generator#40]
there is an option to use mapped field name into the database?

**Comment by marcjulian** [comment]
Do you mean `@map(...)`?

```prisma
model User {
  id        Int      @id @default(autoincrement())
  createdAt DateTime @default(now()) @db.DateTime(0)
  updatedAt DateTime @updatedAt @map("updated_at") 👈
  email     String   @unique
  name      String?  @db.VarChar(255)
}
```

Last time I checked https://github.com/notiz-dev/prisma-dbml-generator/issues/18#issuecomment-1072154314 only `@@map(...)` was included in the DMMF schema from prisma.

**Comment by erlanggadewa** [comment]
yes sir, can you add @map(...) in DMFF schema from prisma ? 
thank you

**Comment by marcjulian** [comment]
@erlanggadewa if you want `@map(...)` to be included in the DMMF you need to open an issue in the [prisma](https://github.com/prisma/prisma) repo. I will closes this as it is the same as #18.
